infrastructure/network/netadapter/router/router.go

```diff
@@ -24,10 +24,10 @@ type Router struct {
 }
 
 // NewRouter creates a new empty router
-func NewRouter() *Router {
+func NewRouter(name string) *Router {
 	router := Router{
 		incomingRoutes: make(map[appmessage.MessageCommand]*Route),
-		outgoingRoute:  newRouteWithCapacity("", outgoingRouteMaxMessages),
+		outgoingRoute:  newRouteWithCapacity(name, outgoingRouteMaxMessages),
```

Comment:
Can you change it to `fmt.Sprintf("%s - outgoing route")` or something like this, so we can know this route is an outgoing route?

Reply:
This will mean then that every route created via NewRoute() is the outgoing one, right?

Reply:
Done